### DISCUSSION TITLE
Support for beancount v3

### DISCUSSION THREAD

**Discussion by geoff-mdt** [ananthakumaran/paisa#300] [General]
Hi 

Paisa is a very promising dashboard for managing one's personal finance. I use beancount for mine and it has been updated since mid-2024 to v3 which rendered some previous features obsolete. For example, bean-extract has been depreciated from the latest realease. 

v3 is now the default and recommanded version of beancount : [Announcement: v3 and git branches - Martin Blais](https://groups.google.com/g/beancount/c/iTdRuvZnE4E/m/sdQk_2__BQAJ)

Would it be possible for paisa to fully support the latest version of beancount ?

**Comment by azzamsa** [comment]
I use Beancount v3 and I have no issue using it with Paisa.

**Comment by vnwnv** [comment]
As #331 mentioned, paisa can not run normally.

The beancount removed the `bean-report` https://github.com/beancount/beancount/issues/832 and paisa can not start and analyze the beancount file.
